src/modules/keyboardmanager/KeyboardManagerEditorLibrary/LoadingAndSavingRemappingHelper.cpp

```diff
@@ -88,6 +89,11 @@ namespace LoadingAndSavingRemappingHelper
             // If they are mapped to the same key, delete those entries and set the common version
             if (table[leftKey] == table[rightKey])
             {
+                if (std::holds_alternative<DWORD>(table[leftKey]) && std::get<DWORD>(table[leftKey]) == combinedKey)
+                {
+                    // Avoid mapping a key to itself when the combined key is equal to the resulting mapping.
+                    return;
+                }
```

Comment:
How can we get this case? We don't allow `X (Left/Right) -> X`.

Reply:
If it's already there on the saved settings from older versions, we can get this.